### PR TITLE
graphviz 3.0.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
>   ==> /opt/homebrew/Cellar/plantuml/1.2022.1/bin/plantuml -testdot
  Picked up _JAVA_OPTIONS: -Duser.home=/Users/brew/Library/Caches/Homebrew/java_cache -Djava.io.tmpdir=/private/tmp
  Dot version: dot - graphviz version 3.0.0 (20220226.1711)
  Warning : cannot determine dot version

**Comment by cho-m** [comment]
`plantuml` looks for 2-digit minor versions, so current code is incompatible with `graphviz 3.x` releases.

https://github.com/plantuml/plantuml/blob/v1.2022.1/src/net/sourceforge/plantuml/cucadiagram/dot/GraphvizVersionFinder.java#L75
```java
		final Pattern p = Pattern.compile("\\d\\.\\d\\d");
```

**Comment by carlocab** [comment]
plantuml/plantuml#940

File: Formula/plantuml.rb [hunk 1]
```diff
@@ -15,12 +18,16 @@ class Plantuml < Formula
     sha256 cellar: :any_skip_relocation, all: "cd5a6877e5b41eacf286bdb9c425730c0a045c9fa4de612a287566da2d9b6415"
   end
 
+  depends_on "ant" => :build # remove when switching to Jar url
   depends_on "graphviz"
   depends_on "openjdk"
 
+  patch :DATA
```
Comment (carlocab) [inline Formula/plantuml.rb:25]: This seems to work. Depending on how reliable their `-testdot` is, this may be good to merge.

**Review by carlocab — APPROVED** [review]
Not opposed to just merging this. If someone complains that `plantuml` is broken we can just `resource` the right `graphviz` version.

**Comment by cho-m** [comment]
Sure, that is fine with me.

---

Not too important note: in-file patch has some tricky whitespace since the SourceForge tarball uses `dos` format (GitHub repo uses `unix`, so patches are not compatible). Some carriage returns are there that aren't visible on browser. Just need to watch out if editing as some editors may auto-convert on save.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**